src/lib/operators/join_hash.cpp

```diff
@@ -644,17 +644,20 @@ class JoinHash::JoinHashImpl : public AbstractJoinOperatorImpl {
 
     const auto l2_cache_size = 256'000;  // bytes
 
-    // We assume an std::unordered_map with a linked list within the buckets.
     // To get a pessimistic estimation (ensure that the hash table fits within the cache), we assume
-    // that each value maps to two RowIDs.
+    // that each value maps to a PosLis with a single RowID. For the used small_vector's, we assume a
```

Comment:
typo

Reply:
I'll fix it. What do you think about that estimation?

Reply:
Looks good on first view.